src/UI/Album/Summary/AlbumSummaryLayout.js

```diff
@@ -0,0 +1,119 @@
+var reqres = require('../../reqres');
+var Marionette = require('marionette');
+var Backgrid = require('backgrid');
+var TrackFileModel = require('../../Artist/TrackFileModel');
+var TrackFileCollection = require('../../Artist/TrackFileCollection');
+var FileSizeCell = require('../../Cells/FileSizeCell');
+var QualityCell = require('../../Cells/QualityCell');
+var DeleteEpisodeFileCell = require('../../Cells/DeleteEpisodeFileCell');
+var NoFileView = require('./NoFileView');
+var LoadingView = require('../../Shared/LoadingView');
+
+module.exports = Marionette.Layout.extend({
+    template : 'Album/Summary/AlbumSummaryLayoutTemplate',
+
+    regions : {
+        overview : '.album-overview',
+        activity : '.album-file-info'
+    },
+
+    columns : [
+        {
+            name     : 'path',
+            label    : 'Path',
+            cell     : 'string',
+            sortable : false
+        },
+        {
+            name     : 'size',
+            label    : 'Size',
+            cell     : FileSizeCell,
+            sortable : false
+        },
+        {
+            name     : 'quality',
+            label    : 'Quality',
+            cell     : QualityCell,
+            sortable : false,
+            editable : true
+        },
+        {
+            name     : 'this',
+            label    : '',
+            cell     : DeleteEpisodeFileCell,
+            sortable : false
+        }
+    ],
+
+    templateHelpers : {},
+
+    initialize : function(options) {
+        if (!this.model.artist) {
+            this.templateHelpers.artist = options.artist.toJSON();
+        }
+    },
+
+    onShow : function() {
+        if (this.model.get('hasFile')) { //TODO Refactor for Albums
```

Comment:
We have to figure out how to check this for albums, as only tracks will have the hasFile property